### PR TITLE
Add PEI and Standalone MM AARCH64 binaries [Rebase & FF]

### PR DESCRIPTION
**Pull request by makubacki**
## Description

---

**Add PEI and Standalone MM AARCH64 binaries**

Produces an AARCH64 CryptoPei and CryptoStandaloneMm binary.

Note:

- Uses the `MmServicesTableLib` and `StandaloneMmDriverEntryPoint` library
  instances in `StandaloneMmPkg` for `AARCH64`.
- Uses `BaseRngLibTimerLib` instance for `AARCH64 MM_STANDALONE` due to current
  platform compatibility.
- Uses `RngDxe` instance for `DXE` drivers so platforms can publish a
  `gEdkiiCryptoProtocolGuid` protocol instance backed by an `RngLib`
  (or alternative) implementation of choice.
- `AARCH64` binaries can be built for all flavors using GCC or Visual Studio,
  however, only GCC binaries are currently published and supported.

---

**AARCH64 GCC Build and Pipeline Support**

- Add pipeline support for AARCH64 GCC builds
- Build AARCH64 GCC binaries in the Mu Devops Ubuntu container
- Update MultiFlavorBuild.py to force AARCH64 builds to use GCC.

---

- [x] Impacts functionality?
  - **Functionality** - Does the change ultimately impact how firmware functions?
  - Examples: Add a new library, publish a new PPI, update an algorithm, ...
- [x] Impacts security?
  - **Security** - Does the change have a direct security impact on an application,
    flow, or firmware?
  - Examples: Crypto algorithm change, buffer overflow fix, parameter
    validation improvement, ...
- [ ] Breaking change?
  - **Breaking change** - Will anyone consuming this change experience a break
    in build or boot behavior?
  - Examples: Add a new library class, move a module to a different repo, call
    a function in a new library class in a pre-existing module, ...
- [ ] Includes tests?
  - **Tests** - Does the change include any explicit test code?
  - Examples: Unit tests, integration tests, robot tests, ...
- [ ] Includes documentation?
  - **Documentation** - Does the change contain explicit documentation additions
    outside direct code modifications (and comments)?
  - Examples: Update readme file, add feature readme file, link to documentation
    on an a separate Web page, ...

## How This Was Tested

- Local build of all crypto flavors
- Test on QEMU Mu Tiano Platform packages

## Integration Instructions

- Use the AARCH64 PEI and Standalone MM binaries on a platform if needed.

### CODE REVIEW

**Review by kuqin12 — APPROVED** [review]

**Review by apop5 — APPROVED** [review]

**Comment by makubacki** [comment]
@kenlautner is going to update the pipeline for GCC. I will update submodules to their tip when that is done, before completing the PR.

**Comment by makubacki** [comment]
> @kenlautner is going to update the pipeline for GCC. I will update submodules to their tip when that is done, before completing the PR.

Done

File: MultiFlavorBuild.py [hunk 1]
```diff
@@ -98,23 +98,49 @@ def Go(self, WorkSpace, PackagesPath, PInHelper, PInManager):
...
-                    if self.stop:
-                        break
+                arches = []
+                use_gcc = False
+                for arch in self.arch:
+                    if arch != "AARCH64":
+                        arches.append(arch)
+                    else:
+                        use_gcc = True
+                if (len(arches) > 0):
```
Comment (makubacki) [inline MultiFlavorBuild.py:108]: Going to drop the parentheses here.

**Review by Javagedes — APPROVED** [review]

File: MultiFlavorBuild.py (hunk 1 above)
Comment (makubacki) [reply MultiFlavorBuild.py]: Done

**Pull request merged**